sql/catalyst/src/main/scala/org/apache/spark/sql/types/DataTypes.java

```diff
@@ -15,77 +15,74 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.api.java;
+package org.apache.spark.sql.types;
```

Comment:
Since this is a java source file, should we move it to `sql/catalyst/src/main/java` ?